docs/en/user-guide/chat/managing-tasks.md

```
@@ -25,18 +26,19 @@ This guide explains how to create conversation-type tasks in the Wegent frontend
 In Wegent, a conversation is the primary way users interact with AI agents. Each conversation creates a Task that records the complete conversation history.
 
 **Core Concept**:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)